.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: build
+
+on: [push]
```

Comment:
I would probably only do it on PR and not push - what I've seen on other projects is that it can cause a backlog of runs on the free GH workers and ends up delaying the PR's. That said we don't have a lot of work going on this repo so maybe it's ok.

Reply:
We have set a max-timeout of 120 mins. Even if something like that happens in the worst case, the github runner gets killed after 120 mins and will be freed.

Reply:
As we don't push out any executable or image from this repo anyways removing it makes sense @akhilerm right?

Reply:
Removed